### ISSUE TITLE
[macos][v.2.12.4] No '-b' option for top

### ISSUE THREAD

**Issue by rprimus** [AirVPN/airvpn-client#52]
Mon Mar 13 21:59:53 GMT 2017

``top`` on MacOS does not have a ```-b``` (batch option) as  found on ```linux```.

From the log file:

```
2017.03.13 21:53:13 - Shell of 'sh','-c 'top -b -n 1 | awk '{print $1,$12}''' done sync in 45 ms, Output: invalid option or syntax: -b
. 2017.03.13 21:53:13 - top usage: top
. 2017.03.13 21:53:13 - [-a | -d | -e | -c <mode>]
. 2017.03.13 21:53:13 - [-F | -f]
. 2017.03.13 21:53:13 - [-h]
. 2017.03.13 21:53:13 - [-i <interval>]
. 2017.03.13 21:53:13 - [-l <samples>]
. 2017.03.13 21:53:13 - [-ncols <columns>]
. 2017.03.13 21:53:13 - [-o <key>] [-O <secondaryKey>]
. 2017.03.13 21:53:13 - [-R | -r]
. 2017.03.13 21:53:13 - [-S]
. 2017.03.13 21:53:13 - [-s <delay>]
. 2017.03.13 21:53:13 - [-n <nprocs>]
. 2017.03.13 21:53:13 - [-stats <key(s)>]
. 2017.03.13 21:53:13 - [-pid <processid>]
. 2017.03.13 21:53:13 - [-user <username>]
. 2017.03.13 21:53:13 - [-U <username>]
. 2017.03.13 21:53:13 - [-u]
. 2017.03.13 21:53:13 - awk: syntax error at source line 1
. 2017.03.13 21:53:13 - context is
. 2017.03.13 21:53:13 - >>>  <<<
. 2017.03.13 21:53:13 - awk: illegal statement at source line 1
. 2017.03.13 21:53:13 - missing }
. 2017.03.13 21:53:13 - Shell of 'sh','-c 'ping -c 1 -t 3 -q -n 194.187.251.114'' done sync in 76 ms, Output: PING 194.187.251.114 (194.187.251.114): 56 data bytes
. 2017.03.13 21:53:13 - --- 194.187.251.114 ping statistics ---
```

**Comment by Clodo76** [comment]
Confirmed. Will be replaced with "ps -eo pid,command" in the next release. Thanks.

**Comment by rprimus** [comment]
Fri Jun  9 16:49:38 BST 2017

Fixed in ```2.13.1```
